coralproject/talk
# Create Plugins - Create plugins through the CLI

# Create Plugins - Create plugins through the CLI

## What does this PR do?
- Allows you to create plugins through the CLI
- Fixes bug having plugins with empty config or without slots

## How do I test this PR?
- Pull the branch `git checkout create-plugin`
- Go to the console and type `./bin/cli-plugins create`
- Answer the questions _**HINT**: Choose yes on client ;)_
- It should look like this: <img width="569" alt="screen shot 2017-07-09 at 12 13 36 pm" src="https://user-images.githubusercontent.com/1401559/27995173-6ba43a18-64a0-11e7-9a24-44865e56ceb6.png">

- Go to to Talk `localhost:3000`
- Refresh the page and you should see a brand new plugin on the stream.
<details>
<summary>**SPOILER**: Your new plugin: </summary>
<img width="560" alt="screen shot 2017-07-09 at 12 17 07 pm" src="https://user-images.githubusercontent.com/1401559/27995223-e639a66e-64a0-11e7-85fa-2b39fb780f27.png">
</details>
